projects/plugins/jetpack/_inc/client/at-a-glance/backups.jsx

```diff
@@ -102,6 +102,71 @@ class DashBackups extends Component {
 		} );
 	};
 
+	getJetpackBackupBanner() {
+		const { partnerCoupon, upgradeUrl, siteRawUrl, trackUpgradeButtonView } = this.props;
+
+		if ( this.props.hasConnectedOwner ) {
+			if ( partnerCoupon ) {
+				const checkoutUrl = getRedirectUrl( 'jetpack-plugin-partner-coupon-checkout', {
+					path: partnerCoupon.product.slug,
+					site: siteRawUrl,
+					query: `coupon=${ partnerCoupon.coupon_code }`,
+				} );
+
+				return (
+					<JetpackBanner
+						callToAction={ __( 'Redeem', 'jetpack' ) }
+						title={ sprintf(
+							/* translators: %s: Name of a Jetpack product. */
+							__(
+								'Redeem your coupon and get started with %s for free the first year!',
+								'jetpack'
+							),
+							partnerCoupon.product.title
+						) }
+						disableHref="false"
+						href={ checkoutUrl }
+						eventFeature="backups"
+						path="dashboard"
+						plan={ getJetpackProductUpsellByFeature( FEATURE_SITE_BACKUPS_JETPACK ) }
+						trackBannerDisplay={ trackUpgradeButtonView }
```

Comment:
Should we change this event to be coupon specific? This will trigger a view event with a feature value of `backup`, so we won't be able to calculate a click rate (views vs clicks).

Note: I assume we have a click event as well - from memory I'm 85% sure we do

Reply:
I've updated the data tracking part of the PR accordingly.